pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -292,6 +293,18 @@ func buildHTTPVirtualServices(
 			vs.Route = route
 		}
 
+		if r.Timeouts != nil {
+			if r.Timeouts.Request != nil {
+				vs.Timeout = durationpb.New(r.Timeouts.Request.Duration)
+			}
+			if r.Timeouts.BackendRequest != nil {
+				if vs.Retries == nil {
+					vs.Retries = &istio.HTTPRetry{Attempts: 1}
+				}
+				vs.Retries.PerTryTimeout = durationpb.New(r.Timeouts.BackendRequest.Duration)
```

Comment:
```suggestion
				timeout := durationpb.New(r.Timeouts.BackendRequest.Duration)
				if vs.Retries != nil {
					vs.Retries.PerTryTimeout = timeout
				} else {
					vs.Timeout = timemout
				}
```

Reply:
addressed in a6c6566e5675613b5ea7215f0c4d9546a952aa34